lib/format/datetime/formatter.ex

```diff
@@ -94,10 +94,10 @@ defmodule Timex.Format.DateTime.Formatter do
     "#{hour}:#{minute}:#{sec}#{ms}"
   end
   def format_token(token, %DateTime{} = date, modifiers, _flags, _width)
-    when token in [:iso_8601, :iso_8601z] do
+    when token in [:iso_8601_extended, :iso_8601_extended_z] do
```

Comment:
Ah, sorry. I wasn't able to figure out what part of the interface was public to maintain compatibility. I restored the `iso_8601` functions and added the extended ones as supplementary.